### PR TITLE
add calendar feed

### PR DESCRIPTION
**Pull request by lsf37**
- adds a Google calendar feed for foundation events and dates, incl hangouts
- update hangout dates. We might reduce these to just the next date in the future now that the calendar exists.

### CODE REVIEW

**Comment by lsf37** [comment]
It's also possible to embed the calendar in html (via iframe), but it doesn't look very nice, so I've not done that.

File: content/contact/home.pml [hunk 1]
```diff
@@ -42,31 +43,32 @@ agenda, anybody can join for technical discussions about seL4, related
 repositories and projects, or general questions and answers. Members from the
 technical steering committee (TSC) will try to be there, but these are not
 TSC meetings where formal decisions are made.<br>
-The next sessions are:
+A <a href="https://calendar.google.com/calendar/u/0?cid=NXNybzFhaGtodDNsb2NuaGRxMnFmbDhndGNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ">calendar
+feed</a> is available. The next sessions are:
```
Comment (axel-h) [inline content/contact/home.pml:47]: We should delete this dates here and only refer to the feed from now on.

Comment (axel-h) [reply content/contact/home.pml:47]: ok

**Comment by axel-h** [comment]
> It's also possible to embed the calendar in html (via iframe), but it doesn't look very nice, so I've not done that.

However, it is convenient to see it (in "agenda" view mode")

**Review by axel-h — APPROVED** [review]
It's good to be merged as a step forward

**Comment by lsf37** [comment]
> However, it is convenient to see it (in "agenda" view mode")

Yes that one could be nice. Does it pick a time zone automatically based on client time zone?

**Comment by axel-h** [comment]
> > However, it is convenient to see it (in "agenda" view mode")
> 
> Yes that one could be nice. Does it pick a time zone automatically based on client time zone?

It seems so, I have removed the timezone param and it still shows my local time.

**Pull request merged**